tidepods/vhr_imdfile.py

```diff
@@ -615,6 +615,8 @@ def main(infile, level, outfolder = None, resolution= None, date=None, timestamp
   
     mikepath = os.environ['MIKE'] = "C:\Program Files (x86)\DHI"
     #ikepath = os.environ.get['MIKE']
+    mikepath = os.environ['MIKE'] = "C:\Program Files (x86)\DHI"
```

Comment:
you still setting the env variable here. remove it and just get it as you did in the other files